dockerfiles/packaging/debian/package/etc/james/jmap.properties

```diff
@@ -0,0 +1,13 @@
+# Configuration file for JMAP
+
+enabled=true
+
+tls.keystoreURL=file://conf/keystore
```

Comment:
the issue is not that it's not starting, the issue is that it is starting with the same keystore for every user, which is very bad imho